pnpm-lock.yaml

```diff
@@ -0,0 +1,8370 @@
+lockfileVersion: "9.0"
```

Comment:
nit: haha I use `pnpm` as well, but in this case it's better to avoid comitting the lock IMO, since this codebase is supposed to be used with `npm`